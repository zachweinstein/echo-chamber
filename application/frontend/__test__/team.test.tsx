// File: __tests__/team.test.jsx
import  TeamPage from '~/pages';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react'


jest.mock('next/router', () => ({
	useRouter() {
		return { pathname: '/' };
	},
}));
 
it('renders homepage unchanged', () => {
	const { container } = render(<TeamPage />)
	expect(container).toMatchSnapshot()
  })


  describe('team member list', () => {
	it('renders a heading', () => {
	  render(<TeamPage />)   
	})
	
  })
