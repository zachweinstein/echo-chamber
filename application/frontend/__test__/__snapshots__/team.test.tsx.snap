// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex-1"
  >
    <div
      class="container relative"
    >
      <section
        class="mx-auto flex max-w-[980px] flex-col items-center gap-2 py-8 md:py-12 md:pb-8 lg:py-24 lg:pb-20"
      >
        <h1
          class="text-center text-3xl font-bold leading-tight tracking-tighter md:text-6xl lg:leading-[1.1]"
        >
          Emergent behavior on social media
        </h1>
        <p
          class="max-w-[750px] text-center text-lg text-muted-foreground sm:text-xl"
        >
          Build prompts, experiment on multiple models, and witness the emergence of dynamic interactions between language models.
        </p>
        <div
          class=" mt-2"
        >
          <a
            class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-white transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-zinc-950 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 dark:ring-offset-zinc-950 dark:focus-visible:ring-zinc-300 bg-zinc-900 text-zinc-50 hover:bg-zinc-900/90 dark:bg-zinc-50 dark:text-zinc-900 dark:hover:bg-zinc-50/90 h-10 px-4 py-2"
            href="/login"
          >
            Get Started
          </a>
        </div>
      </section>
    </div>
  </div>
</div>
`;
